File: base64/ruby/encode_decode.rb
#!/usr/bin/env ruby

require "base64"

if ARGV.length < 1
  puts "Please specify filename"
  exit 1
end

filename = ARGV[0]

f1 = File.new(filename, 'r')
bin1 = f1.read
st1 = Base64.encode64(bin1)

bin2 = Base64.decode64(st1)
f2 = File.new(filename + ".new", "w")
f2.write(bin2)
f2.close

File: base64/python/encode_decode.py
#!/usr/bin/env python

import base64
import sys

if len(sys.argv) < 2:
    print "Please specify filename"
    sys.exit(1)

filename = sys.argv[1]
f1 = open(filename, 'r')
bin1 = f1.read()
st = base64.encodestring(bin1)

bin2 = base64.decodestring(st)
f2 = open(filename + ".new", "w")
f2.write(bin2)
f1.close()
f2.close()







